examples/some-verifier/frontend/src/components/Issuer.tsx

```diff
@@ -30,12 +30,17 @@ interface DiscordWindowMessage {
 // This is set when Discord verification is started and read upon a message back
 let oAuth2State: string | undefined = undefined;
 
-function Issuer() {
-  const query = useMemo(() => new URLSearchParams(window.location.search), []);
-  const telegram = query.get(Platform.Telegram);
-  const discord = query.get(Platform.Discord);
-  const [telegramDone, setTelegramDone] = useState(telegram === 'true');
-  const [discordDone, setDiscordDone] = useState(discord === 'true');
+function Issuer({
+  telegramIssued,
+  setTelegramIssued,
+  discordIssued,
+  setDiscordIssued,
+}: {
+  telegramIssued: boolean;
+  setTelegramIssued: () => void;
+  discordIssued: boolean;
+  setDiscordIssued: () => void;
+}) {
```

Comment:
Would prefer not to inline types 🙂 

```suggestion
interface IssuerProps {
  telegramIssued: boolean;
  setTelegramIssued: () => void;
  discordIssued: boolean;
  setDiscordIssued: () => void;
}

function Issuer({
  telegramIssued,
  setTelegramIssued,
  discordIssued,
  setDiscordIssued,
}: IssuerProps) {
```